src/govuk/components/notification-banner/notification-banner.yaml

```diff
@@ -0,0 +1,134 @@
+params:
+- name: title
+  type: string
+  required: false
+  description: Custom title text. Defaults to 'Important' ('Success' for success banner and 'Error' for error banner). If `titleHtml` is supplied, this is ignored.
+- name: titleHtml
+  type: string
+  required: false
+  description: Custom title HTML. Defaults to 'Important' ('Success' for success banner and 'Error' for error banner).
+- name: titleHeadingLevel
+  type: string
+  required: false
+  description: Heading level, from 1 to 6. Default is `2`.
+- name: content
+  type: string
+  required: true
+  description: Content for the notification banner. If `contentHtml` is supplied, this is ignored.
+- name: contentHtml
+  type: string
+  required: true
+  description: HTML content for the notification banner.
+- name: type
+  type: string
+  required: false
+  description: Sets classes and attributes to create a type of notification banner. Accepts `success` or `alert`.
```

Comment:
Perhaps we could add something brief to the description about what this actually affects? E.g: the role and visual styling of the banner?

Reply:
I've tried to improve this documentation and will review it with Eoin next.

Reply:
The tech docs are going to be reviewed by Eoin so will resolve this for now.